### PR TITLE
[SIG-2900] LocationMarker component

### PR DESCRIPTION
**Pull request by janjaap**
This PR adds the `LocationMarker` component. This component doesn't actually render anything, because it taps directly into the Leaflet API to add vector layers to the map instance.

I tried using both `Circle` and `CircleMarker` from [Amsterdam/react-maps](https://github.com/Amsterdam/amsterdam-react-maps), but I ran into issues with the zoom buttons not being clickable or the `Circle` vector scaling unpredictably when zooming in or out.

### CODE REVIEW

File: src/components/LocationMarker/__tests__/LocationMarker.test.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import L from 'leaflet';
```
Comment (jpoppe) [inline src/components/LocationMarker/__tests__/LocationMarker.test.js:1]: Any reason to use a one letter variable name here?

File: src/components/LocationMarker/index.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import L from 'leaflet';
```
Comment (jpoppe) [inline src/components/LocationMarker/index.js:3]: Same here

File: src/components/LocationMarker/__tests__/LocationMarker.test.js (hunk 1 above)
Comment (janjaap) [reply src/components/LocationMarker/__tests__/LocationMarker.test.js:1]: That's how it's been done so far where Leaflet was included. I didn't think twice about it. I can change it, if you want me to.

**Comment by janjaap** [comment]
Import renamed

**Pull request merged**